mapca/tests/test_utils.py

```diff
@@ -95,7 +95,7 @@ def test_kurtn():
     """
     Unit test for _kurtn function
     """
-    test_data = np.random.rand(2, 3, 4)
+    test_data = np.random.rand(2, 3)
     kurt = _kurtn(test_data)
     assert kurt.shape == (3, 1)
```

Comment:
This is a smoke test (i.e., checks that it ran and gave the right kinds of outputs, but not the right values). What about writing a unit test (known inputs and known outputs for a few possible configurations) to make sure that the results are the same?

Reply:
Yes! I wanted to do that. I was thinking that once the integration test works, we could use the inputs and results of each function to write the unit tests.

Reply:
I removed the test because it will no longer be inside `utils` and we will be using the `scipy` function anyway (which should work).

Reply:
Nevermind, I'll add the test. `scipy` could change the function any time so better safe than sorry.

Reply:
I think that test is a great idea. It won't impact coverage, but one that directly compares the original code (hardcoded into the test or stored as a test utils function perhaps) would be perfect.

Reply:
Great idea! Just pushed the changes with such test.